spec/features/visitors/email_confirmation_spec.rb

```diff
@@ -0,0 +1,132 @@
+require 'rails_helper'
+
+feature 'Email confirmation during sign up' do
+  scenario 'confirms valid email and sets valid password' do
+    email = 'test@example.com'
+    sign_up_with(email)
+    raw_confirmation_token, = Devise.token_generator.generate(User, :confirmation_token)
+    User.find_by(email: email).update(
+      confirmation_token: raw_confirmation_token, confirmation_sent_at: Time.current
+    )
+
+    visit "/users/confirmation?confirmation_token=#{raw_confirmation_token}"
+
```

Comment:
alrighty -- updated at a3a1274 to use some super cool methods from email_spec that are new to me! `visit_in_email` is the method for clicking email links :) https://github.com/email-spec/email-spec/blob/70bd7709ff3b505e339ef68f1ae9c9de926495c3/spec/email_spec/helpers_spec.rb